sdks/go/examples/wordcount/wordcount.go

```diff
@@ -95,10 +95,12 @@ var (
 
 // Concept #3: You can make your pipeline assembly code less verbose by
 // defining your DoFns statically out-of-line. A DoFn can be defined as a Go
-// function and is conventionally suffixed "Fn". The argument and return types
-// dictate the pipeline shape when used in a ParDo. For example,
+// function and is conventionally suffixed "Fn". Using named function
+// transforms allows for easy reuse, modular testing, and an improved monitoring
+// experience. The argument and return types of a function dictate the pipeline
+// shape when used in a ParDo. For example,
 //
-//	formatFn: string x int -> string
+//	func formatFn(w string, c int) string
```

Comment:
We should make extractFn match below (sorry I didn't call that out initially)

Reply:
Okay, I think it's equivalent now...